cmd/lncli/commands.go

```diff
@@ -2485,41 +2487,36 @@ func sendToRoute(ctx *cli.Context) error {
 			len(routes.Routes))
 	}
 
-	req := &lnrpc.SendToRouteRequest{
+	req := &routerrpc.SendToRouteRequest{
 		PaymentHash: rHash,
 		Route:       routes.Routes[0],
 	}
 
 	return sendToRouteRequest(ctx, req)
 }
 
-func sendToRouteRequest(ctx *cli.Context, req *lnrpc.SendToRouteRequest) error {
-	client, cleanUp := getClient(ctx)
-	defer cleanUp()
+func sendToRouteRequest(ctx *cli.Context, req *routerrpc.SendToRouteRequest) error {
+	conn := getClientConn(ctx, false)
+	defer conn.Close()
 
-	paymentStream, err := client.SendToRoute(context.Background())
+	client := routerrpc.NewRouterClient(conn)
+
+	resp, err := client.SendToRoute(context.Background(), req)
 	if err != nil {
 		return err
 	}
 
-	if err := paymentStream.Send(req); err != nil {
-		return err
+	marshaller := jsonpb.Marshaler{
+		Indent: "    ",
 	}
 
-	resp, err := paymentStream.Recv()
+	var buf bytes.Buffer
+	err = marshaller.Marshal(&buf, resp)
 	if err != nil {
-		return err
+		fatal(err)
 	}
 
-	printJSON(struct {
-		E string       `json:"payment_error"`
-		P string       `json:"payment_preimage"`
-		R *lnrpc.Route `json:"payment_route"`
-	}{
-		E: resp.PaymentError,
-		P: hex.EncodeToString(resp.PaymentPreimage),
-		R: resp.PaymentRoute,
-	})
+	fmt.Println(buf.String())
```

Comment:
We used the prior custom JSON here in order to properly print out the pre-image. Otherwise, it'll show up as hex encoded.

Reply:
Yes, but that encoding doesn't show for example the enum String() values. This is just a debug commit that will be taken off before merge.

Reply:
Gotcha, I didn't see any indication that it wasn't mean to be a part of the final PR. I think it can safely be removed now?

Reply:
The commit msg is marked with `[debug]`.

Reply:
I will remove it after approved reviews. Until that time it is useful for testing.